portfolio/src/main/webapp/script.js

```diff
@@ -56,4 +48,109 @@ function openTab(evt, tabName) {
   evt.currentTarget.className += " active";
 }
 
+function createMap() {
+  const map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
Nit: it's fine to keep this now, but should consider to move this to a config file in the future.